README.md

```diff
@@ -6,45 +6,79 @@ This software is distributed under the terms of the Apache License,
 Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
 <!-- ../../okapi/doc/md2toc -l 2 -h 4 README.md -->
-* [Introduction](#introduction)
-* [Compiling](#compiling)
-* [Docker](#docker)
-* [Installing the module](#installing-the-module)
-* [Deploying the module](#deploying-the-module)
-* [Additional information](#additional-information)
-
-## Introduction
-
-The module provides the role of delivering messages using SMTP server to send email.
-This module use `mod-configuration` to get connection parameters.
-
-The module supports the following configuration for SMTP server:
-
- |  PARAMETERS              |  DESCRIPTION                           |  EXAMPLES                      |
- |--------------------------|----------------------------------------|--------------------------------|
- |  EMAIL_SMTP_HOST         |  the hostname of the smtp server       | 'localhost'                    |
- |  EMAIL_SMTP_PORT         |  the port of the smtp server           | 502                            |
- |  EMAIL_SMTP_LOGIN_OPTION |  the login mode for the connection     | DISABLED, OPTIONAL or REQUIRED |
- |  EMAIL_TRUST_ALL         |  trust all certificates on ssl connect | true or false                  |
- |  EMAIL_SMTP_SSL          |  sslOnConnect mode for the connection  | true or false                  |
- |  EMAIL_START_TLS_OPTIONS |  TLS security mode for the connection  | DISABLED, OPTIONAL or REQUIRED |
- |  EMAIL_USERNAME          |  the username for the login            | 'login'                        |
- |  EMAIL_PASSWORD          |  the password for the login            | 'password'                     |
- |  EMAIL_FROM              |  'from' property of the email          | noreply@folio.org              |
- |  AUTH_METHODS            |  authentication methods                | 'CRAM-MD5 LOGIN PLAIN'         |
-
-
- Required configuration options:
-  * EMAIL_SMTP_HOST
-  * EMAIL_SMTP_PORT
-  * EMAIL_USERNAME
-  * EMAIL_PASSWORD
-  * EMAIL_FROM
-  * EMAIL_SMTP_SSL
-
- The main name of the module : "SMTP_SERVER"
-
- Module configuration example:
+
+## Gaol
```

Comment:
@alexanderkurash -- "Gaol" is english word for "place of confinement" (i.e. "Jail" in US english). The word "Goal" would be better.

(I found this because the dev.f.o script that extracts "introduction" descriptions from READMEs suddenly lost it. It looks for a section named "Introduction" or "Goal" etc.)